userspace/libscap/engine/bpf/scap_bpf.c

```diff
@@ -513,6 +517,7 @@ static int32_t load_tracepoint(struct bpf_engine* handle, const char *event, str
 	int err;
 	int fd;
 	int id;
+	const char *prog_name;
```

Comment:
I know it is super useless but it is future-proof :laughing:!

```suggestion
	const char *prog_name = NULL;
```